### PR TITLE
Add piwheels.org to python

### PR DESCRIPTION
**Pull request by henry-y-c**

### CODE REVIEW

File: data/python [hunk 1]
```diff
@@ -4,3 +4,4 @@ pypi.io
 pypi.org
 python.org
 pythonhosted.org
+piwheels.org
```
Comment (IceCodeNew) [inline data/python:7]: > piwheels is a community project by Ben Nuttall and Dave Jones.

I think this domain should be included in another category, or at least add a comment to separate the official projects and the community projects.

**Review by IceCodeNew — CHANGES_REQUESTED** [review]

File: data/python (hunk 1 above)
Comment (henry-y-c) [reply data/python:7]: what do you think we add a new file, maybe named as `python-community`?

Comment (IceCodeNew) [reply data/python:7]: > what do you think we add a new file, maybe named as `python-community`?

I'm OK with that ;-)

Comment (henry-y-c) [reply data/python]: Done

**Review by IceCodeNew — APPROVED** [review]

**Pull request merged**